Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM sdp-docker-registry.kat.ac.za:5000/docker-base-build as build
-MAINTAINER Bruce Merry "bmerry@ska.ac.za"
+LABEL maintainer="sdpdev@ska.ac.za"
```

Comment:
So you can try sdpdev+katsdpcam2telstate@ska.ac.za which should now bounce back to you...

Reply:
Done